po/fr.po

```diff
@@ -1047,9 +1042,10 @@ msgid "No item selected!"
 msgstr "Aucun élément sélectionné !"
 
 #: src/itemlistformaction.cpp:108
-#, fuzzy
 msgid "Do you really want to delete all articles (y:Yes n:No)? "
-msgstr "Souhaitez-vous réellement écraser `%s' (y : Oui n : No)? "
+msgstr ""
+"Souhaitez-vous réellement supprimer tous les articles "
+"`%s' (y : Oui n : No)? "
```

Comment:
The original string doesn't contain `%s`; are you sure the translation is correct?